### PR TITLE
Added civilian buildings.

### PR DESCRIPTION
**Pull request by Dzierzan**

### CODE REVIEW

File: mods/e2140/content/neutral/civ_building/rules.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+^CoreCivilianBuilding:
```
Comment (michaeldgg2) [inline mods/e2140/content/neutral/civ_building/rules.yaml:1]: Lots of duplication, why not inherit from `^CoreBuilding` and disable unnecessary stuff?

File: mods/e2140/content/neutral/civ_building/civ_building3/sequences.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+neutral_building_civ_3:
+	Defaults:
+		Filename: building_civ_building_3.vspr
```
Comment (michaeldgg2) [inline mods/e2140/content/neutral/civ_building/civ_building3/sequences.yaml:3]: Civ building 2 and 3 have the same visual (in `SPRB0.MIX`). Are we sure they do in the original game too?

**Review by michaeldgg2 — APPROVED** [review]

File: mods/e2140/content/neutral/civ_building/civ_building3/sequences.yaml (hunk 2 above)
Comment (Dzierzan) [reply mods/e2140/content/neutral/civ_building/civ_building3/sequences.yaml:3]: `SPRB0.MIX` does not lie.

File: mods/e2140/content/neutral/civ_building/rules.yaml (hunk 1 above)
Comment (Dzierzan) [reply mods/e2140/content/neutral/civ_building/rules.yaml:1]: I would have made it simpler if ORA could null conditions. If you inherit a condition, you cannot make it null again.

Comment (michaeldgg2) [reply mods/e2140/content/neutral/civ_building/rules.yaml:1]: You need to use the `-RequiresCondition:` syntax:

```
	WithDecoration:
		-RequiresCondition:
```
(originally `RequiresCondition: VehicleDisabled` inherited from `^DisabledActor`)

image

Comment (Dzierzan) [reply mods/e2140/content/neutral/civ_building/rules.yaml:1]: Interesting

**Review by michaeldgg2 — APPROVED** [review]

**Pull request merged**